### ISSUE TITLE
Include keys and foreign keys in name.interface.ts

### ISSUE THREAD

**Issue by eddyystop** [feathers-plus/generator-feathers-plus#56]
Like
```js
export interface UserBase {
  _id: unknown;
  foreignKey : uinknown;
}

export interface User extends UserBase {
  _id: any; // change if needed
  foreignKey: any // change if needed
}
```

**Comment by eddyystop** [comment]
```js

// Define TypeScript interface for service `nedb2`. (Can be re-generated.)
// !code: imports // !end
// !code: init // !end

// tslint:disable-next-line:no-empty-interface
export interface Nedb2Base {
  // !<DEFAULT> code: interface
  _id: unknown;
  nedb1Id: unknown;
  // !end
}

// tslint:disable-next-line:no-empty-interface
export interface Nedb2 extends Nedb2Base {
  // !<DEFAULT> code: more
  _id: any; // change if needed
  nedb1Id: any; // change if needed
  // !end
}

// !code: funcs // !end
// !code: end // !end
```

**Comment by eddyystop** [comment]
Published as f+/cli v0.7.34 using generator v0.6.33 .

**Comment by eddyystop** [comment]
@murbanowicz ^^^
